src/utils.ts

```diff
@@ -17,6 +17,12 @@ export const VAULTS = {
       vaultId: 152242747,
       liquidatorDiscount: 0.045,
     },
+    dAlgo:{
```

Comment:
```suggestion
    gAlgo:{
```

Reply:
@AwolDes 
Done, please let me know if you think its good to go to create a new SDK version.